.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,53 @@
+name: Bug Report
+description: Create a report to help us improve
+title: '[BUG]: '
+labels: Type-Defect
+assignees: martinrotter
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Dear RSS Guard contributor, please RESPECT this template. You might be interested in reading [www.chiark.greenend.org.uk](http://www.chiark.greenend.org.uk/~sgtatham/bugs.html).
+        Also, ALWAYS, ALWAYS, ALWAYS attach DEBUG LOG to your bug report!!!
+        How to generate it see [here](https://github.com/martinrotter/rssguard/blob/master/resources/docs/Documentation.md#command-line-interface).
```

Comment:
Because the images are loaded after page was scrolled to id, result is a little bit buggy (I use Firefox on Linux).
I suggest something like "... read 'Command line interface' section of [documentation](https://github.com/martinrotter/rssguard/blob/master/resources/docs/Documentation.md#minor-features)."
What do you think?

Reply:
> result is a little bit buggy (I use Firefox on Linux)

And again it seems to be a problem only of Firefox and only on Linux... I really hate it.

Reply:
> Another one I recently collided with is weird flickering of transition: all ... ...;

No, cross this out. It is caused by one of my add-ons.